src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class RightTriangle implements Figure {
```

Comment:
The class name 'RightTriangle' is appropriate as it is a noun and represents the entity correctly.